experimental/grpc_attestation/src/attestation.rs

```diff
@@ -183,7 +194,7 @@ impl RemoteAttestation for AttestationServer {
         request_stream: Request<tonic::Streaming<AttestedInvokeRequest>>,
     ) -> Result<Response<Self::AttestedInvokeStream>, Status> {
         let tee_certificate = self.tee_certificate.clone();
-        let request_handler = self.request_handler;
+        let request_handler = self.request_handler.clone();
```

Comment:
Why do you need `clone()` if `F` is `static`?

Reply:
`F` has a `'static` lifetime, but the moving and borrowing rules still apply. This is the error without `clone`:
```
cannot move out of `_self.request_handler` which is behind a shared reference
```
I could borrow instead of cloning, but that requires specifying lifetimes everywhere, even if `F` is `'static` in `RequestHandler`.